Hubands and wives transfer.md

```diff
@@ -0,0 +1,27 @@
+# Summary
+>The primary object of the experiment was to estimate the effect of previous exposure to a problem not only on individual, but also on group performance, and thus evaluate how previous experience may influence the relation given by equation
+
+# References
+Main paper: https://onlinelibrary-wiley-com.proxy.library.upenn.edu/doi/pdfdirect/10.1002/bs.3830050103
+
```

Comment:
Tartaglia Problem- book cited in paper: https://books.google.com/books?hl=en&lr=&id=Vm9U6EIJSDgC&oi=fnd&pg=PP1&dq=mathematics+and+imagination&ots=owUlDggmZ9&sig=rRiq_yTUBev9_Y1LIPqMM-d1-1o#v=onepage&q=mathematics%20and%20imagination&f=false

Reply:
see page 159